packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-resources/notebook-resources.html

```diff
@@ -0,0 +1,152 @@
+<div>
+    <!--Notebook resources usage case-->
+    <div>
+        <!--Header-->
+        <p class="oui-heading_5 mb-2">
+            <strong
+                data-translate="pci_notebook_add_resources_usecase"
+            ></strong>
+        </p>
+
+        <!--Use cases-->
+        <div class="row">
+            <oui-select-picker
```

Comment:
You can try to use a ng-repeat for the select-picker...

Reply:
I can't, the both items they are completely separate (not in the same array).
And we can't have others models.